frontend/js/controllers/view_overview.js

```diff
@@ -38,7 +38,11 @@ angular
         // open file externally
         $scope.openDocument = function(article) {
             console.log("Open Document: " + article.filepath.value.value);
-            window.open(settings.files_url + article.filepath.value.value);
+            let nameEncode = encodeURIComponent("test_user");
+            let pwEncode = encodeURIComponent("test_password");
+            let split = settings.files_url.split(/\/\//, 2);
+
+            window.open(split[0] + "//" + nameEncode + ":" + pwEncode + "@" + split[1] + article.filepath.value.value);
```

Comment:
Wunderbar, ich das ist natürlich eine viel bessere Lösung als einen zusätzlichen parameter in der settings einzuführen!